lerna.json

```diff
@@ -3,10 +3,5 @@
   "packages": [
     "packages/*"
   ],
-  "version": "1.0.0",
-  "npmClient": "yarn",
-  "useWorkspaces": true,
```

Comment:
Is this line not necessary anymore? Did you run into problems with yarn workspaces?

Reply:
I was able to get past it, but I can't exactly tell you why it works now